README.md

```diff
@@ -129,22 +140,42 @@ your init-file:
 (prescient-persist-mode +1)
 ```
 
-Another popular choice is to combine the sorting abilities of
-[`prescient.el`](https://github.com/raxod502/prescient.el) with the
-filtering of [`orderless`](https://github.com/oantolin/orderless):
+* Your most recent choices are saved, and those are sorted first.
+  After that, your most frequent choices are saved, and those are
+  sorted next. The rest of the candidates are sorted by length. This
+  algorithm turns out to do very well in practice while being fast and
+  not very magical.
+* Your input is split on spaces into subqueries, each of which must
+  match as either a substring, a regexp, or an initialism (e.g. `ffap`
+  matches `find-file-at-point`) in order for a candidate to be
+  included. Again, this algorithm isn't optimal, but it does very well
+  in practice given its simplicity and speed.
+* The part of each candidate that matched your input is highlighted,
+  with the initials of an initialism highlighted in a second color.
 
-```elisp
-(setq selectrum-prescient-enable-filtering nil)
-(selectrum-prescient-mode +1)
+##### Alternative 2: Orderless
+
+Another popular choice for filtering is to use the flexible
+[`orderless`](https://github.com/oantolin/orderless) completion style.
 
+```elisp
 (setq completion-styles '(orderless))
+
+;; Optional performance optimization
+;; by highlighting only the visible candidates.
 (setq orderless-skip-highlighting (lambda () selectrum-is-active))
 (setq selectrum-highlight-candidates-function #'orderless-highlight-matches)
 ```
 
-For more tips and setup help for integration with other packages see
-our
-[wiki](https://github.com/raxod502/selectrum/wiki/Additional-Configuration).
+The candidates are sorted using the default sorting method of
+Selectrum. Afterwards they are filtered and highlighted using the
+`completion-styles`, in this case `orderless`. On top of Orderless, it
+is also possible to use Prescient *only for sorting* by adding:
+
+```elisp
+(setq selectrum-prescient-enable-filtering nil)
+(selectrum-prescient-mode +1)
+```
```

Comment:
Maybe we should also include `(prescient-persist-mode +1)` here? People deciding for orderless might miss it.

Reply:
Oh okay, right. I missed it :laughing: